src/index.js

```diff
@@ -97,7 +104,7 @@ const ApolloClientPlugin = createPlugin({
       const client = new ApolloClient({
         ssrMode: __NODE__ ? true : false,
         link: apolloLinkFrom(links),
```

Comment:
The terminalLink is annoying here, because it could be a wsLink, or an uploadLink, i think it's better if we could keep terminalLink as default, and let users extend it.